src/components/commons/Header.tsx

```diff
@@ -18,6 +18,7 @@ import { getTotalFromCart } from "../../utils/cart/cart";
 import { moneyFormat } from "../../utils/form/formatters";
 import { paymentSubjectTransform } from "../../utils/transformers/paymentTransformers";
 import DrawerDetail from "../Header/DrawerDetail";
+import cartIcon from "../../assets/images/cart.png";
 
```

Comment:
It's much better to use ShoppingCartIcon component exported from '@mui/icons-material' instead of using a low resolution raster png. Plus, the problem about the alt property will not exist anymore
https://mui.com/material-ui/material-icons/?query=ca&selected=ShoppingCart

Reply:
I do not think it is possible to use the icon provided by material-icons @infantesimone @MarcoAGreco confirmed ?